### PR TITLE
Issue #6037: resolve inspection violation 'Abstract class without abstract method

### PR DESCRIPTION
**Pull request by romani**
Issue #6037

### CODE REVIEW

**Comment by romani** [comment]
> I think it is wrong that the abstract class defines getPackageLocation.

good catch.
fixed

File: src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionCyclomaticComplexityTest.java [hunk 1]
```diff
@@ -30,12 +30,18 @@
 
 public class XpathRegressionCyclomaticComplexityTest extends AbstractXpathTestSupport {
 
+    private final String checkName = CyclomaticComplexityCheck.class.getSimpleName();
```
Comment (rnveach) [inline src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionCyclomaticComplexityTest.java]: I'm surprised nothing is complaining that these fields can be static.

**Comment by rnveach** [comment]
Everything else looks fine to me.

File: src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionCyclomaticComplexityTest.java (hunk 1 above)
Comment (romani) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionCyclomaticComplexityTest.java]: I thought about it, persuaded myself to not do this, as thetr will be no benefits as tests are used in strict workflow, make it static will force it to be uppercased, so getter will return constant, ...

Comment (rnveach) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionCyclomaticComplexityTest.java]: All the other getters in src/test return a constant too.

**Pull request merged**